Rakefile

```diff
@@ -227,3 +227,39 @@ namespace :publish do
     run! "rm -rf BraintreeDropIn.framework.zip"
   end
 end
+
+def jazzy_command
+  %W[jazzy
+      --objc
+      --author Braintree
+      --author_url http://braintreepayments.com
+      --github_url https://github.com/braintree/braintree-ios-drop-in
+      --github-file-prefix https://github.com/braintree/braintree-ios-drop-in/tree/#{current_version}
+      --theme fullwidth
+      --output #{current_version}
+      --xcodebuild-arguments --objc,BraintreeDropIn-Umbrella-Header.h,--,-x,objective-c,-isysroot,$(xcrun --sdk iphonesimulator --show-sdk-path),-I,$(pwd)
+  ].join(' ')
+end
+
+desc "Generate documentation via jazzy and push to GH"
+task :docs_publish => %w[docs:generate docs:publish]
+
+namespace :docs do
+
+  desc "Generate docs with jazzy"
+  task :generate do
+    run(jazzy_command)
+    run! "cp -R Images #{current_version}/Images" # copy images used in README
+    puts "Generated HTML documentation"
+  end
+
+  task :publish do
+    run! "git checkout gh-pages"
+    run! "ln -sfn #{current_version} current" # update symlink to current version
```

Comment:
I structured things like the [docs for the JS SDK](https://github.com/braintree/braintree-web/tree/gh-pages) with a folder for each SDK version and a `current` directory that is just a symlink to the folder for the most recent version.